steps/assets/base/tectonic.tf

```diff
@@ -50,6 +50,7 @@ module "bootkube" {
   service_account_private_key_pem = "${local.service_account_private_key_pem}"
 
   etcd_endpoints = "${data.template_file.etcd_hostname_list.*.rendered}"
+  worker_ign_config = "${file("worker.ign")}"
```

Comment:
Who/what creates the `worker.ign` file? What is inside the file?

Reply:
I see, most likely `generateIgnConfigStep` is responsible for that, right?

Reply:
correct :)